src/material/stepper/stepper.ts

```diff
@@ -102,7 +102,10 @@ export class MatStepper extends CdkStepper implements AfterContentInit {
   @Output() readonly animationDone: EventEmitter<void> = new EventEmitter<void>();
 
   /** Whether ripples should be disabled for the step headers. */
-  @Input() disableRipple: boolean;
+  @Input()
+  get disableRipple(): boolean { return this._disableRipple; }
+  set disableRipple(value: boolean) { this._disableRipple = coerceBooleanProperty(value); }
+  private _disableRipple = false;
```

Comment:
I tried to use `mixinDisableRipple` but VE tests doesn't seem to be happy with this change. Let me know if you're okay with this or how I can proceed with the failing tests.

Reply:
What was the error you were getting?